ql/src/java/org/apache/hadoop/hive/ql/exec/vector/expressions/AbstractFilterStringColLikeStringScalar.java

```diff
@@ -226,7 +226,7 @@ protected interface CheckerFactory {
   protected static final class NoneChecker implements Checker {
     final byte [] byteSub;
 
-    NoneChecker(String pattern) {
+    public NoneChecker(String pattern) {
```

Comment:
@ryukobayashi, not a big deal, but that is not required.

Reply:
@deniskuzZ If it is not `public`, an access violation will result in an exception.
https://github.com/apache/hive/blob/ceafcc411fe3471019a05f2509b5d7a6b242f371/ql/src/java/org/apache/hadoop/hive/ql/exec/vector/expressions/FilterStringColLikeStringScalar.java#L38-L39

Reply:
oh yes, sorry about that